GOVERNANCE.md

```diff
@@ -1,74 +1,22 @@
-# in-toto governance
-This document covers the project's governance and committer process.  The
-project consists of the in-toto
-[specification](https://github.com/in-toto/docs) and
-[reference implementation](https://github.com/in-toto/in-toto).
-
-## Code of Conduct
-
-The in-toto community abides by the Cloud Native Computing Foundation's [code of conduct](/CODE-OF-CONDUCT.md). An excerpt follows:
-
-> _As contributors and maintainers of this project, and in the interest of fostering an open and
-> welcoming community, we pledge to respect all people who contribute through reporting issues,
-> posting feature requests, updating documentation, submitting pull requests or patches, and other
-> activities._
-
-in-toto community members represent the project and their fellow contributors. We value our 
-community tremendously, and we'd like to keep cultivating a friendly and collaborative environment 
-for our contributors and users. We want everyone in the community to have a positive experience.
-
-
-## Maintainership and Consensus Builder
-The project is maintained by the people indicated in
-[MAINTAINERS.txt](MAINTAINERS.txt).  A maintainer is expected to (1) submit and
-review GitHub pull requests and (2) open issues or [submit vulnerability
-reports](https://github.com/in-toto/in-toto#security-issues-and-bugs).
-A maintainer has the authority to approve or reject pull requests submitted by
-contributors.  Any maintainer may also interact with the CNCF on behalf of the
-project.  
-
-The project's Consensus Builder (CB), who is also granted all maintainer 
-privileges and responsibilities, is Santiago Torres-Arias 
-<santiagotorres@purdue.edu, @SantiagoTorres>.
+# in-toto Governance
 
+in-toto's
+[governance](https://github.com/in-toto/community/blob/main/GOVERNANCE.md) and
+[code of conduct](https://github.com/in-toto/community/blob/main/CODE-OF-CONDUCT.md)
+are described in the [in-toto/community](https://github.com/in-toto/community)
+repository.
 
 ## Reference Implementation Contributions
```

Comment:
I think we should move these to a CONTRIBUTING file, as suggested in [GH community standards docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), so that new contributors can easily find them. Especially now that they are no longer referenced in the README.

Reply:
Added a note back to readme. I'd like to keep it consistent with the other repos and have just one file as they're all pointers to in-toto/community anyway.

Reply:
Okay, makes sense. You could also create [default community health files](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) in an `github.com/in-toto/.github` repo, which all just point to in-toto/community.